docs/pages/iconography.js

```diff
@@ -0,0 +1,402 @@
+          },
+        }}
+      >
+        {image}
+      </Box>
+      <Heading size="400">{heading}</Heading>
+      <Box marginBottom={6}>
+        <Text>{text}</Text>
+      </Box>
+    </Flex>
+  );
+}
+
+export default function IconographyPage(): Node {
+  return (
+    <Page title="Iconography">
+      <PageHeader
+        name="Iconography"
+        description={`Our typographic system creates a content hierarchy that is scannable and efficient.
+    `}
+        type="guidelines"
+      />
+
+      <MainSection name="Principles">
+        <Flex gap={12} alignContent="between" wrap>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={275} maxWidth="45%">
+            <PrincipleItem
+              color="purple-mysticool-100"
+              image={<Obvious />}
+              heading="Obvious"
+              text={
+                <Text>
+                  Simple, intuitive, and easily recognizable. Each icon is consolidated to its
+                  minimal form, expressing fundamental aspects. Icons should feel balanced, and
+                  align elements optically.
+                </Text>
+              }
+            />
+          </Flex.Item>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={275} maxWidth="45%">
+            <PrincipleItem
+              color="green-matchacado-100"
+              image={<Intentional />}
+              heading="Intentional"
+              text={
+                <Text>
+                  Icons should be used purposefully for their intended meaning of supporting good
+                  comprehension. If questioning an icon’s use, it probably doesn’t need to be used
+                  at all.
+                </Text>
+              }
+            />
+          </Flex.Item>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={275} maxWidth="45%">
+            <PrincipleItem
+              color="blue-skycicle-100"
+              image={<Accessible />}
+              heading="Accessible"
+              text={
+                <Text>
+                  We aim to a shape that ensures readability and clearness, even at small sizes. Our
+                  icons should be understandable by a global audience of users, regardless of
+                  nationality or language.
+                </Text>
+              }
+            />
+          </Flex.Item>
+        </Flex>
+      </MainSection>
+
+      <MainSection
+        name="Typography components"
+        description={`
+Some of our React components use icons as primary elements to call attention to a particular action or information.
+
+- [IconButton](/icon): Uses icons instead of text to convey available actions on a screen.
+- [Status](/status): A graphic indicator of an element’s state.
+- [Link](/link): External links use the visit icon as an external domain indicator.
+`}
+      />
+
+      <MainSection name="Characteristics">
+        <MainSection.Subsection title="Appearance">
+          <Markdown
```

Comment:
I don't remember so apologies if we've discussed this, but can this content go in the "description" prop of Subsection instead? It should still be rendered in markdown

Reply:
I added it this way as the content is intermixed w/ an image so it felt strange to have one block in a `description` prop and then another as a `Markdown` component. Thoughts?